p2panda-js/test/index.test.ts

```diff
@@ -15,4 +15,14 @@ describe('KeyPair', () => {
     const keyPairSecond = KeyPair.fromPrivateKey(keyPair.privateKey());
     expect(keyPair.publicKey()).to.eq(keyPairSecond.publicKey());
   });
+
+  it('signs and validates', async () => {
+    const { KeyPair } = await p2panda;
+    const keyPair = new KeyPair();
+    const message = 'hello panda';
+    const sig = keyPair.sign(message);
+    expect(keyPair.verify(message, sig)).to.be.true;
+    expect(() => keyPair.verify(message, sig)).to.not.throw();
+    expect(() => keyPair.verify('hello aquadoggo', sig)).to.throw();
```

Comment:
We were talking about this earlier. Would it be more javascriptonic to make this

```
    expect(keyPair.verify('wrong message', sig)).to.be.false;
```

instead of throwing?

Reply:
Yeah, I agree, it should only throw when something is wrong which would make validation impossible :+1:

Reply:
ok, but this is not about the atomic data types, i'll pull it out

Reply:
wait.. actually we implemented the key pair thing also in this pr. so i'll add it here